prompt_generator.py

```diff
@@ -100,4 +108,25 @@ def get_prompt(current_start_date, current_end_date, ou_id):
 I am including a sample Email below for you to refer. This is only for reference, you can alter the email structure if need be. 
 
 {sample_email}
-""".format(prompt_body=prompt_body, prompt_header=prompt_header, prompt_tail=prompt_tail, sample_email=sample_email)
+""".format(prompt_body=prompt_body, prompt_header=prompt_header, prompt_tail=prompt_tail, sample_email=sample_email),jsonl_data
+
+def chart_generator(data):
+    data = read_jsonl("/Users/yogeshmaheshwari/Desktop/AutoQBR/aqbr/data.jsonl")
```

Comment:
Do not read from local file. You need to fetch from metbase and read. See how it has been done previously.

Reply:
Removed